doc/changelog/09-cli-tools/19863-coqdep-nom.rst

```diff
@@ -0,0 +1,4 @@
+- **Removed:**
+  `coqdep` flag `-m` (it was used through `coq_makefile`)
```

Comment:
Actually, the flag was also used by people who manually call `coqdep`, e.g., to produce a graph of dependencies such as https://coqinterval.gitlabpages.inria.fr/interval/

Reply:
That should probably be using -I like the coqc calls.